tools/functional-tester/etcd-runner/command/lock_racer_command.go

```diff
@@ -69,12 +72,16 @@ func runRacerFunc(cmd *cobra.Command, args []string) {
 		m := concurrency.NewMutex(s, racers)
 		rcs[i].acquire = func() error { return m.Lock(ctx) }
 		rcs[i].validate = func() error {
+			mu.Lock()
```

Comment:
OK nevermind the mutex isn't being applied to the election. Is this the right fix, though? Why is a follower `acquire`ing the election-- shouldn't acquire mean exclusive ownership?